mesheryctl/internal/cli/root/system/restart.go

```diff
@@ -94,6 +94,16 @@ func restart() error {
 			}
 			log.Info("Restarting pod ", name)
 		}
+
+		ok, err := utils.IsMesheryRunning(currPlatform)
+		if err != nil {
+			return err
+		}
+		if !ok {
+			if err := start(); err != nil {
+				return errors.Wrap(err, utils.SystemError("Failed to restart Meshery"))
+			}
+		}
```

Comment:
can this be an if...else? If Meshery is running then great, restart the pods. If Meshery is not running, let's start Meshery and don't attempt to restart the pods.

Reply:
@navendu-pottekkat yes, good idea. Just check my latest commit once to see if we're on the same page...